psscriptanalyzer/Test-Connection.ps1

```
@@ -0,0 +1,52 @@
+    }
+}
+
+function Test-Endpoints {
+    $urls = @(
+        "https://demoapp.example.com",
+        "https://api.demoapp.example.com",
+        "https://admin.demoapp.example.com"
+    )
+    foreach ($u in $urls) {
+        $resp = iwr -Uri $u -UseBasicParsing
+        if ($resp.StatusCode -eq $null) {
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)